ocaml/xapi/nm.ml

```diff
@@ -613,18 +613,17 @@ let bring_pif_up ~__context ?(management_interface = false) (pif : API.ref_PIF)
                     let addresses =
                       List.filter_map
                         (fun addr_and_prefixlen ->
-                          try
-                            let n = String.index addr_and_prefixlen '/' in
+                          let ( let* ) l r =
+                            Option.bind (Result.to_option l) r
```

Comment:
I haven't seen this before, but is this a good way of using the let binding (as opposed to making it explicit)

Reply:
Yeah, there were two usages of these unconventional `let*` during development but now that there's one I will make it explicit.